packages/desktop-client/src/redux/index.ts

```diff
@@ -0,0 +1,6 @@
+import { useDispatch, useSelector } from 'react-redux';
+
+import { type AppDispatch, type RootState } from 'loot-core/client/store';
+
+export const useAppDispatch = useDispatch.withTypes<AppDispatch>();
+export const useAppSelector = useSelector.withTypes<RootState>();
```

Comment:
Is it possible to keep the names the same so we don't have to rename everywhere?

```suggestion
import { useDispatch as useReduxDispatch, useSelector as useReduxSelector } from 'react-redux';

import { type AppDispatch, type RootState } from 'loot-core/client/store';

export const useDispatch = useReduxDispatch.withTypes<AppDispatch>();
export const useSelector = useReduxSelector.withTypes<RootState>();
```

Reply:
I have reverted these renames. These are based on redux toolkit recommendations https://redux.js.org/usage/usage-with-typescript. Will separate them out on another PR instead to make this PR more manageable to review.